<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAlarms</class>
 <widget class="QWidget" name="FormAlarms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_alarms">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Alarmes</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_logs">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Log de Eventos</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageAlarms">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="lst_alarms">
         <property name="font">
          <font>
           <pointsize>28</pointsize>
          </font>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="spacing">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="ButtonBeR" name="btn_return">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>100</height>
            </size>
           </property>
           <property name="text">
            <string>Retornar</string>
           </property>
           <property name="icon">
            <iconset resource="../res/icons.qrc">
             <normaloff>:/icons/arrow_backward.png</normaloff>:/icons/arrow_backward.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="bgIcon">
            <iconset resource="../res/icons.qrc">
             <normaloff>:/icons/bt-blank.png</normaloff>
             <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
           </property>
           <property name="bgIconSize">
            <size>
             <width>80</width>
             <height>80</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="ButtonBeR" name="btn_reset_alarms">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>110</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>110</height>
            </size>
           </property>
           <property name="text">
            <string>Resetar
Alarmes</string>
           </property>
           <property name="icon">
            <iconset resource="../res/icons.qrc">
             <normaloff>:/icons/cancel.png</normaloff>:/icons/cancel.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="bgIcon">
            <iconset resource="../res/icons.qrc">
             <normaloff>:/icons/bt-blank.png</normaloff>
             <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
           </property>
           <property name="bgIconSize">
            <size>
             <width>80</width>
             <height>80</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageLogs">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QListWidget" name="lst_logs">
         <property name="font">
          <font>
           <family>Monospace</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1,0">
         <property name="spacing">
          <number>18</number>
         </property>
         <item>
          <widget class="ButtonBeR" name="btn_minus_month">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../res/icons.qrc">
             <normaloff>:/icons/arrow-left.png</normaloff>:/icons/arrow-left.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="bgIcon">
            <iconset resource="../res/icons.qrc">
             <normaloff>:/icons/bt-blank.png</normaloff>
             <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
           </property>
           <property name="bgIconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_date">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>48</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(85, 85, 85);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>JANEIRO</string>
           </property>
           <property name="maxLength">
            <number>16</number>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Normal</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ButtonBeR" name="btn_plus_month">
           <property name="minimumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../res/icons.qrc">
             <normaloff>:/icons/arrow-right.png</normaloff>:/icons/arrow-right.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="bgIcon">
            <iconset resource="../res/icons.qrc">
             <normaloff>:/icons/bt-blank.png</normaloff>
             <normalon>:/icons/bt-blank_pressed.png</normalon>:/icons/bt-blank.png</iconset>
           </property>
           <property name="bgIconSize">
            <size>
             <width>48</width>
             <height>48</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCalendarWidget" name="calendarWidget">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>280</height>
          </size>
         </property>
         <property name="gridVisible">
          <bool>true</bool>
         </property>
         <property name="horizontalHeaderFormat">
          <enum>QCalendarWidget::NoHorizontalHeader</enum>
         </property>
         <property name="verticalHeaderFormat">
          <enum>QCalendarWidget::NoVerticalHeader</enum>
         </property>
         <property name="navigationBarVisible">
          <bool>false</bool>
         </property>
         <property name="dateEditEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonBeR</class>
   <extends>QPushButton</extends>
   <header>buttonber.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/icons.qrc"/>
 </resources>
 <connections/>
</ui>
